src/app/pages/server-compare/server-compare.component.scss

```
@@ -150,6 +150,11 @@
   }
 }
 
+:host ::ng-deep .block_full_inline.memory-compare-chart--cache-markers {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)